graphrag_sdk/fixtures/prompts.py

```diff
@@ -461,6 +461,19 @@
 Question: {question}
 """
 
+CYPHER_GEN_PROMPT_WITH_HISTORY = """
+Using the ontology provided, generate an OpenCypher statement to query the graph database returning all relevant entities, relationships, and attributes to answer the question below.
+
+First, determine if the last answer provided to the user is relevant to the current question. If it is relevant, you may consider incorporating information from it into the query. If it is not relevant, ignore it and generate the query based solely on the question.
```

Comment:
Don't restrict the agent from looking at only the last answer, especially when the entire chat history is available to it.